hack/common.sh

```diff
@@ -39,5 +39,11 @@ function compile() {
   pack_build ${completion_image} "./cmd/completion"
   completion_image=${resolved_image_name}
 
-  ytt -f config/. -v controller_image=${controller_image} -v webhook_image=${webhook_image} -v build_init_image=${build_init_image} -v rebase_image=${rebase_image} -v completion_image=${completion_image} > $output
+  ytt -f config/. \
+    -v controller_image=${controller_image} \
+    -v webhook_image=${webhook_image} \
+    -v build_init_image=${build_init_image} \
+    -v rebase_image=${rebase_image} \
+    -v completion_image=${completion_image} \
+    -v lifecycle_image="gcr.io/cf-build-service-public/lifecycle-0.5.0" > $output
```

Comment:
Where does this image come from? Can we avoid needing to build a lifecycle image to our toolchain by just pointing it at the lifecycle layer in an existing image. After all, it just needs to be a layer. 

Like this layer?`cloudfoundry/cnb@sha256:8f0b2d09ab4b38530a1630403967d11a601e56e02e79d3f56370d34fd071fe38`

Reply:
We could use remote.Layer("cloudfoundry/cnb@sha256:8f0b2d09ab4b38530a1630403967d11a601e56e02e79d3f56370d34fd071fe38") to construct it.

Reply:
I think this makes sense. I like the idea of not having to maintain a new image. I do worry that it is confusing to just pull a layer from another image, but the savings probably out-weigh the confusion factor.

Reply:
While looking into this somemore today, Danny and I realized that it probably makes more sense to use a lifecycle image instead of a lifecycle layer, as images are distributable artifacts in a way that layers aren't. Additionally, we can attached lifecycle metadata to the lifecycle image but that wouldnt be possible in the case of a layer. I agree that this involves additional work on our end to maintain that image, but I think there are advantages to this approach outweigh referencing a layer